specification/billing/resource-manager/Microsoft.Billing/preview/2018-08-01-preview/billing.json

```diff
@@ -0,0 +1,1371 @@
+      "get": {
+        "tags": [
+          "Invoices"
+        ],
+        "operationId": "Invoices_GetLatest",
+        "description": "Gets the most recent invoice. When getting a single invoice, the downloadUrl property is expanded automatically.  This is only supported for Azure Web-Direct subscriptions. Other subscription types which were not purchased directly through the Azure web portal are not supported through this preview API.",
+        "x-ms-examples": {
+          "InvoicesGetLatest": {
+            "$ref": "./examples/InvoicesGetLatest.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/Invoice"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_List",
+        "description": "Lists all of the available billing REST API operations.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/projects/{projectId}/initiateImportRequest": {
+      "post": {
+        "description": "Initiates the request to import the GTM or legacy subscriptions or RIs to GTM.",
+        "operationId": "ImportRequest_Initiate",
+        "x-ms-examples": {
+          "InitiateImportRequest": {
+            "$ref": "./examples/InitiateImportRequest.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/billingAccountIdParameter"
+          },
+          {
+            "$ref": "#/parameters/projectIdParameter"
+          },
+          {
+            "$ref": "#/parameters/initiateImportRequestParameter"
+          }
+        ],
+        "tags": [
+          "AIPController"
+        ],
+        "responses": {
+          "200": {
```

Comment:
Shouldn't this be a 202? Based on the presence of a status field in the response payload, I assume that the request may succeed or fail after the response is returned. 'Accepted' seems to be a more accurate response in this case.

Reply:
Import request operations defined in this swagger by themselves are not long running.
Like initiating the import request is not running operation, though actual import of product might take longer. So to track this import of product status there is status field provided in response.